### DISCUSSION TITLE
Set route query parameters when making a request

### DISCUSSION THREAD

**Discussion by motion-work** [Diizzayy/nuxt-graphql-client#282] [Q&A] [answered]
Is it possible to pass route query parameters when making a post request to the graphql endpoint?
something like: `https://myapi.com/graphql?token=34543545436`

**Comment by Diizzayy** [comment, marked as answer]
@motion-work the [`useGqlHost` composable](https://nuxt-graphql-client.web.app/getting-started/composables#usegqlhost) might be what you're looking for.

**Reply by motion-work** [reply]
exactly what I was looking for. thanks a lot!
